src/E/E_RandomDistribution.cpp

```diff
@@ -12,7 +12,7 @@
 namespace E
 {
 
-RandomDistribution::RandomDistribution() : engine(time(0))
+RandomDistribution::RandomDistribution() : engine(rand())
```

Comment:
this is good

Reply:
This will let you not to change other files except making `testenv` to set `srand` properly.